packages/x-components/src/x-modules/query-suggestions/wiring.ts

```diff
@@ -118,5 +128,8 @@ export const querySuggestionsWiring = createWiring({
   },
   UserClickedOutOfMainModal: {
     clearQuerySuggestionsQuery
+  },
+  UserAcceptedAQueryPreview: {
+    setQuerySuggestionsQueryPreview
   }
```

Comment:
As in RTs and Popular Searches modules, we need to update the `extraParams` here:

```suggestion
  UserAcceptedAQueryPreview: {
    setQuerySuggestionsQueryPreview,
    setQuerySuggestionsParamsFromQueryPreview
  }
```
 and add the commit above: 

`export const setQuerySuggestionsParamsFromQueryPreview = wireCommit(
  'setParams',
  ({ eventPayload: { extraParams } }) => extraParams
);`

Test: select a query preview and reopen empathize: 
![Captura de pantalla 2023-08-29 a les 15 31 12](https://github.com/empathyco/x/assets/21217131/2040a717-127c-4c00-aab2-2d5497cb0fa1)